op-program/host/flags/flags.go

```diff
@@ -83,13 +83,14 @@ var (
 	}
 	L2Claim = &cli.StringFlag{
 		Name:    "l2.claim",
-		Usage:   "Claimed L2 output root to validate",
+		Usage:   "Claimed L2 root to validate",
 		EnvVars: prefixEnvVars("L2_CLAIM"),
 	}
 	L2BlockNumber = &cli.Uint64Flag{
 		Name:    "l2.blocknumber",
-		Usage:   "Number of the L2 block that the claim is from",
-		EnvVars: prefixEnvVars("L2_BLOCK_NUM"),
+		Aliases: []string{"l2.sequencenumber"},
+		Usage:   "Number of the L2 block number or timestamp that the claim is from",
```

Comment:
```suggestion
		Usage:   "L2 block number or timestamp that the claim is from",
```

Reply:
oh wow, that was a terrible effort on my part...